src/puppetlabs/puppetdb/scf/storage_utils.clj

```diff
@@ -278,12 +295,38 @@ must be supplied as the value to be matched."
   (when value
     (json/parse-string (.getValue value))))
 
+(pls/defn-validated str->pgobject :- PGobject
```

Comment:
Since the `pgobject->str` is really just `(.getValue ...)` I didn't really see the need since it would really just be a renaming.